dissect/hypervisor/disk/vmdk.py

```diff
@@ -398,18 +402,69 @@ def __getattr__(self, attr):
         return getattr(self.hdr, attr)
 
 
+RE_EXTENT_DESCRIPTOR = re.compile(
+    r"""
+    ^
+    (?P<access_mode>RW|RDONLY|NOACCESS)\s
+    (?P<sectors>\d+)\s
+    (?P<type>SPARSE|ZERO|FLAT|VMFS|VMFSSPARSE|VMFSRDM|VMFSRAW)
+    (\s?(?P<filename>\".+\")\s?)?
```

Comment:
```suggestion
    (\s(?P<filename>\".+\"))?
```

The first space probably shouldn't be optional?
The final space can be picked up by match part for the next field I guess, as it only is of consequence if there is a next field. Or is it possible to have spaces appended at the end of the line (without a next field)?
Similar for the next 3 lines.